elements/pl-string-input/pl-string-input.py

```diff
@@ -47,8 +47,19 @@ def render(element_html, data):
         editable = data['editable']
         raw_submitted_answer = data['raw_submitted_answers'].get(name, None)
 
+        if remove_leading_trailing:
+            if remove_spaces:
+                space_hint = 'Leading, trailing spaces and whitespaces between text will be removed from your answer.'
+            else:
+                space_hint = 'Leading and trailing spaces will be removed from your answer.'
+        else:
+            if remove_spaces:
+                space_hint = 'Whitespaces between text will be removed but leading and trailing spaces will be left as part of your answer.'
```

Comment:
```suggestion
                space_hint = 'All spaces between text will be removed but leading and trailing spaces will be left as part of your answer.'
```

I think that "whitespace" is not a generally understood term 😄